frontend/components/map/component.tsx

```diff
@@ -1,7 +1,6 @@
 import { useEffect, useState, useRef, useCallback, FC } from 'react';
 
 import ReactMapGL, { FlyToInterpolator, TRANSITION_EVENTS } from 'react-map-gl';
```

Comment:
Below this line, there used to be this one, which I think is necessary:

```tsx
import 'maplibre-gl/dist/maplibre-gl.css';
```

Reply:
I've tried with and without the import, it doesn't seem to make a difference.